projects/1.20/assets/project-expansion/projectexpansion/lang/zh_cn.json

```
@@ -0,0 +1,585 @@
+	"command.projectexpansion.book.empty": "未找到任何位置。",
+	"command.projectexpansion.book.click_to_copy": "点击以复制。这可用于提供给开发者进行调试。",
+	"command.projectexpansion.book.bound_to_player.dump": "这本书绑定到玩家 %s。要导出这些位置，请使用 %s。",
+	"command.projectexpansion.book.bound_to_player.list": "这本书绑定到玩家 %s。要列出这些位置，请使用 %s。",
+	"command.projectexpansion.book.bound_to_player.clear": "这本书绑定到玩家 %s。要清除这些位置，请使用 %s。",
+	"command.projectexpansion.book.bound_to_player.add": "这本书绑定到玩家 %s。要添加位置，请使用 %s。",
+	"command.projectexpansion.book.bound_to_player.remove": "这本书绑定到玩家 %s。要移除位置，请使用 %s。",
+	"command.projectexpansion.book.list.location": "名称：%s 位置：%s 维度：%s",
+	"command.projectexpansion.book.clear.itemstack_success": "该书中的位置已被清除。",
+	"command.projectexpansion.book.clear.player_success": "%s 的位置已被清除。",
+	"command.projectexpansion.book.clear.player_success_self": "你的位置已被清除。",
+	"command.projectexpansion.book.clear.player_notification": "你的传送位置已被 %s 清除。",
+	"command.projectexpansion.book.remove.invalid_location": "未找到该名称的位置。",
+	"command.projectexpansion.book.remove.internal_location": "该位置是内部位置，无法移除。",
+	"command.projectexpansion.book.remove.itemstack_success": "该位置已成功从书中移除。",
+	"command.projectexpansion.book.remove.player_success": "该位置已成功从 %s 的传送位置中移除。",
+	"command.projectexpansion.book.remove.player_success_self": "该位置已成功从你的传送位置中移除。",
+	"command.projectexpansion.book.remove.player_notifcation": "位置 \"%s\" 已被 %s 从你的传送位置中移除。",
+	"command.projectexpansion.book.remove.backup": "位置备份：%s",
+	"command.projectexpansion.book.remove.backup_info": "位置的备份，以防误删。",
+	"command.projectexpansion.book.add.duplicate_name": "已存在同名位置。",
+	"command.projectexpansion.book.add.invalid_name": "该名称无效。",
+	"command.projectexpansion.book.add.itemstack_success": "该位置已成功添加到书中。",
+	"command.projectexpansion.book.add.player_success": "该位置已成功添加到 %s 的传送位置中。",
+	"command.projectexpansion.book.add.player_success_self": "该位置已成功添加到你的传送位置中。",
+	"command.projectexpansion.book.add.player_notification": "位置 \"%s\" 已被 %s 添加到你的传送位置中。",
+	"command.projectexpansion.emc.invalid": "你提供的值（\"%s\"）无效，必须是正整数。",
+	"command.projectexpansion.emc.add.success": "成功为 %s 增加了 %s EMC，现在他们拥有 %s EMC。",
+	"command.projectexpansion.emc.add.success_self": "成功为你的EMC增加了 %s，现在你拥有 %s EMC。",
+	"command.projectexpansion.emc.add.notification": "你被 %s 给予了 %s EMC，现在你拥有 %s EMC。",
+	"command.projectexpansion.emc.remove.success": "成功从 %s 的EMC中移除了 %s，现在他们拥有 %s EMC。",
+	"command.projectexpansion.emc.remove.success_self": "成功从你的EMC中移除了 %s，现在你拥有 %s EMC。",
+	"command.projectexpansion.emc.remove.notification": "你的 %s EMC 被 %s 移除，现在你拥有 %s EMC。",
+	"command.projectexpansion.emc.remove.negative": "无法从 %s 中移除 %s EMC，这会使其EMC变为负数。",
+	"command.projectexpansion.emc.set.success": "成功将 %s 的EMC设置为 %s。",
+	"command.projectexpansion.emc.set.success_self": "成功将你的EMC设置为 %s。",
+	"command.projectexpansion.emc.set.notification": "你的EMC已被 %s 设置为 %s。",
+	"command.projectexpansion.emc.get.success_self": "你拥有 %s EMC。",
+	"command.projectexpansion.emc.get.success": "%s 拥有 %s EMC。",
```

```suggestion
	"command.projectexpansion.emc.get.success": "%s拥有%s EMC。",
```
在1.16或更高版本下，英文字母、阿拉伯数字等非中文文本无需在前后添加空格。
具体可见我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **04.排版** 一节下。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。

……不过例如`10000 EMC`这种，数字和单位之间加不加空格随意。汉语环境一般来说是不加的，但是按排版来说加上去更美观